src/palimpzest/dataclasses.py

```diff
@@ -16,10 +16,10 @@ class GenerationStats:
     # raw_answers: Optional[List[str]] = field(default_factory=list)
 
     # the total number of input tokens processed by this operator; None if this operation did not use an LLM
-    total_input_tokens: int = 0.0
+    total_input_tokens: int = 0
```

Comment:
One subtle note on this, I think what we want to do is to change the type here (and in `RecordOpStats`) to `float` and add a second comment between L18-L19 to the effect of: `# total_input_tokens is typed as a float because GenerationStats may be amortized (i.e. divided) across a number of output records` (same goes for `total_output_tokens`)

Reply:
Resolved